src/Views/Form/Themes/Legacy/optionConfig.php

```diff
@@ -25,7 +26,7 @@
 				'attributes'    => [
 					'placeholder' => __( 'Donate Now', 'give' ),
 				],
-				'wrapper_class' => 'give-hidden',
+				'wrapper_class' => '_give_reveal_label_field give-hidden',
```

Comment:
@henryholtgeerts Can you explain the need for this class?

Reply:
@ravinderk This wrapper class is required so that conditional options are hidden/displayed by the admin-forms.js file. Rather than updating that file, I opted to match fields in the legacy theme to classes that file already looks for.